Automattic/wp-calypso
# Site Transfers: Add email sent notice

Closes https://github.com/Automattic/dotcom-forge/issues/2628

## Proposed Changes

This PR adds `Email sent successfully` notice to the UI when the email for the transfer was sent and makes it consistent [with Figma design.](1U910ArebC7gbLF1RtxUxE-fi-1_2)

<img width="887" alt="Screenshot 2023-06-05 at 2 09 55 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/74f1aa47-c800-4980-8c6c-767d401d2d93">

**Before**:

<img width="813" alt="Screenshot 2023-06-05 at 2 11 18 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/c754fa88-69d6-4fd0-a6b1-3163556d1b45">

## Testing Instructions

* Make sure you have a Simple site
* Navigate to `Settings > General > Transfer your site`: [https://wpcalypso.wordpress.com/settings/start-site-transfer/{site}](https://wpcalypso.wordpress.com/settings/start-site-transfer/%7Bsite%7D)
* Enter in the field the username or email of the user to transfer site to and click on "Search user and continue"
* Proceed to the next screen
* Toggle on all the authorizations and click on `Start transfer`
* Proceed to the next screen and observe the `Email sent successully` notice

## Pre-merge Checklist
